draft-ietf-rats-corim.md

```diff
@@ -674,13 +676,10 @@ The following describes each member of the `triples-map`:
 * `coswid-triples` (index 6): Triples associating modules with existing CoSWID tags.
   Described in {{sec-comid-triple-coswid}}.
 
-* `conditional-endorsement-series-triples` (index 8): Triples describing a series of
-  conditional Endorsements based on the acceptance of a stateful environment.
+* `conditional-endorsement-series-triples` (index 8): Triples describing a series of Endorsement that are applicable based on the acceptance of a series of stateful environment records.
   Described in {{sec-comid-triple-cond-series}}.
 
-* `conditional-endorsement-triples` (index 10): Triples describing a series of
-  Endorsement that are applicable based on the acceptance of a series of
-  stateful environment records.
+* `conditional-endorsement-triples` (index 10): Triples describing a series of conditional Endorsements based on the acceptance of a stateful environment.
```

Comment:
We should be consistent in stating either `stateful environment` or `stateful environment records` between `conditional-endorsement-series-triples` and `conditional-endorsement-triples`

Reply:
Should this spec define a process for reclassifying OTI triples as MTI? For example, that a newer RFC could update an older RFC reclassifying from OTI to MTI or deprecating something (or is that obvious)?

Reply:
> We should be consistent in stating either `stateful environment` or `stateful environment records` between `conditional-endorsement-series-triples` and `conditional-endorsement-triples`

I don't understand the significance. Isn't it sufficient to refer to the triples based on the name used in triples-map?

Reply:
Ah CoRIM profile can prescribe its own MTIs. It cannot makre MTI from the base spec OTI, but it can make existing OTI from the base spec MTI. I am not certain how to deal with MTI from profiles, but I assume that a profile can inherit triple semantics from other profiles as is or redefine them MTI or OTI.

Reply:
> A CoRIM profile can prescribe its own MTIs. It cannot makre MTI from the base spec OTI, but it can make existing OTI from the base spec MTI. I am not certain how to deal with MTI from profiles, but I assume that a profile can inherit triple semantics from other profiles as is or redefine them MTI or OTI.

I interpreted Ned's question to be about whether we should create a process here to regulate how MTI and OTI classes are allowed to move.
My answer is no, we don't.  Let the future selves and fellow RATS decide how to deal based on accumulated wisdom.

Reply:
Fair enough